nfs-ganesha/build/build_deb

```diff
@@ -36,15 +44,24 @@ cd build
 # make source tarball
 cmake -DCMAKE_BUILD_TYPE=Maintainer -DUSE_FSAL_GLUSTER=OFF -DUSE_FSAL_CEPH=ON -DUSE_FSAL_RGW=ON $WORKSPACE/nfs-ganesha/src && make dist || exit 1
```

Comment:
yes the tarball `make dist` creates doesn't have a src directory, it is the `src` directory. I can override some of the cpack config variables to make the process a little cleaner looking.